README.md

```diff
@@ -34,9 +34,9 @@ command on a system that has git installed:
 	git clone https://github.com/ceph/ceph.git
 
 When the ceph/ceph repository has been cloned to your system, run the following
-command to check out the git submodules associated with the ceph/ceph
-repository: 
+commands to move into the cloned ceph/ceph repository and to check out the git submodules associated with it :
```

Comment:
I would backtick `ceph/ceph`

Reply:
I was on the fence, but now that I have your vote I am not on the fence.